src/midi/midi_mt32.cpp

```diff
@@ -0,0 +1,516 @@
+	        "  Accepted file names are as follows:\n"
+	        "    MT32_CONTROL.ROM or CM32L_CONTROL.ROM - control ROM file.\n"
+	        "    MT32_PCM.ROM or CM32L_PCM.ROM - PCM ROM file.");
+
+	auto *bool_prop = sec_prop.Add_bool("reverse.stereo", when_idle, false);
+	bool_prop->Set_help("Reverse stereo channels for MT-32 output");
+
+	bool_prop = sec_prop.Add_bool("verbose", when_idle, false);
+	bool_prop->Set_help("MT-32 debug logging");
+
+	bool_prop = sec_prop.Add_bool("thread", when_idle, false);
+	bool_prop->Set_help("MT-32 rendering in separate thread");
+
+	auto *int_prop = sec_prop.Add_int("chunk", when_idle, 16);
+	int_prop->SetMinMax(2, 100);
+	int_prop->Set_help(
+	        "Minimum milliseconds of data to render at once. (min 2, max 100)\n"
+	        "Increasing this value reduces rendering overhead which may improve performance but also increases audio lag.\n"
+	        "Valid for rendering in separate thread only.");
+
+	int_prop = sec_prop.Add_int("prebuffer", when_idle, 32);
+	int_prop->SetMinMax(3, 200);
+	int_prop->Set_help(
+	        "How many milliseconds of data to render ahead. (min 3, max 200)\n"
+	        "Increasing this value may help to avoid underruns but also increases audio lag.\n"
+	        "Cannot be set less than or equal to mt32.chunk value.\n"
+	        "Valid for rendering in separate thread only.");
+
+	int_prop = sec_prop.Add_int("partials", when_idle, 32);
+	int_prop->SetMinMax(8, 256);
+	int_prop->Set_help(
+	        "The maximum number of partials playing simultaneously. (min 8, max 256");
+
+	const char *mt32DACModes[] = {"0", "1", "2", "3", 0};
+	int_prop = sec_prop.Add_int("dac", when_idle, 0);
+	int_prop->Set_values(mt32DACModes);
+	int_prop->Set_help(
+	        "MT-32 DAC input emulation mode\n"
+	        "Nice = 0 - default\n"
+	        "Produces samples at double the volume, without tricks.\n"
+	        "Higher quality than the real devices\n\n"
+
+	        "Pure = 1\n"
+	        "Produces samples that exactly match the bits output from the emulated LA32.\n"
+	        "Nicer overdrive characteristics than the DAC hacks (it simply clips samples within range)\n"
+	        "Much less likely to overdrive than any other mode.\n"
+	        "Half the volume of any of the other modes.\n"
+	        "Perfect for developers while debugging :)\n\n"
+
+	        "GENERATION1 = 2\n"
+	        "Re-orders the LA32 output bits as in early generation MT-32s (according to Wikipedia).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number, and XX means the bit is always low):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 XX\n\n"
+
+	        "GENERATION2 = 3\n"
+	        "Re-orders the LA32 output bits as in later generations (personally confirmed on my CM-32L - KG).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 14");
```

Comment:
To be hidden behind some kind of debug flag, for developer's use only?

Reply:
Yup. Suggest all of these hidden values just live in a config block of constexpr similar to GUS, where anyone can easily adjust and recompile.